testsUtils/testsUtils.md

```diff
@@ -27,20 +27,50 @@ simulate.click(eventHolder, 10, 20);
 
 Provides utility functions for testing the color of a pixel or area from a canvas:
 
-* getPixelColor
-* getScaledPixelColor
-* rgba
-* isClose
-* canvasData;
-* getEntireCanvasData;
-* hexToRgb;
+* getPixelColor(ctx, x, y)
 
-A Jasmine matcher to test a particular pixel color of a plot is also provided:
+Returns the color of a pixel, specified by a canvas context and its xy position.
 
+* getScaledPixelColor(ctx, r, x, y)
```

Comment:
What is the difference between this function and getPixelColor>

Reply:
lost description while arranging items :(